src/components/sidebar/tabs/ModelLibrarySidebarTab.vue

```diff
@@ -212,6 +222,52 @@ watch(
   },
   { deep: true }
 )
+
+let dropTargetCleanup = () => {}
+
+onMounted(() => {
+  dropTargetCleanup = dropTargetForElements({
+    element: document.body,
```

Comment:
Do we really want the drop target to be the whole document? This approach won't give us room if later we want to highlight the dropzone and droppable state.

I would suggest we put this on-hold and redesign the drag and drop system in `app.ts` first.
https://github.com/Comfy-Org/ComfyUI_frontend/blob/2b265141902fa7f0e7804a9e725be78448553152/src/scripts/app.ts#L1015-L1048

Ideally whether a node can accept the drop of an embedding should be defined on that particular node's declaration.